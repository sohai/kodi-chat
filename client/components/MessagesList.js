import React from 'react';
import { pure, compose } from 'recompact';
import PropTypes from 'prop-types';
import classNames from 'classnames';
import Message from './Message';

const propTypes = PropTypes && {};
const defaultProps = {};

export const MessagesListComponent = ({ items }) => (
  <div className="content">
    {items.map((item, idx) => <Message key={idx} item={item} />)}
  </div>
);

MessagesListComponent.displayName = 'MessagesList';
MessagesListComponent.propTypes = propTypes;
MessagesListComponent.defaultProps = defaultProps;
export default compose(pure)(MessagesListComponent);
