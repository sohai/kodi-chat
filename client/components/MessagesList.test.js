/* global shallow, render, describe, it, expect */
/* eslint-disable import/first */
import React from 'react';
import { MuiThemeProvider } from 'material-ui/styles';
import theme from '../../theme/theme';
import MessagesList from './MessagesList';

describe('(Component) MessagesList', () => {
  it('should render', () => {
    const props = {
      children: <span>test</span>
    };
    const wrapper = render(
      <MuiThemeProvider theme={theme}>
        <MessagesList {...props} />
      </MuiThemeProvider>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
