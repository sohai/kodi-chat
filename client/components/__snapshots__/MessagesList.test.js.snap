// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) MessagesList should render 1`] = `
<div
  class="content"
/>
`;
